gpkit/constraints/prog_factories.py

```diff
@@ -144,10 +144,10 @@ def solve_pass(i):
         raise RuntimeWarning("no sweeps solved successfully.")
 
     solution["sweepvariables"] = KeyDict()
-    ksweep, klinkedsweep = KeyDict(sweep), KeyDict(linkedsweep)
+    ksweep = KeyDict(sweep)
     delvars = set()
     for var, val in solution["constants"].items():
-        if var in ksweep or var in klinkedsweep:
```

Comment:
linked variables no longer require a sweep! you can have a constant that is a function of other constants whether or not you are sweeping.